src/main/resources/languages/en-US.json

```diff
@@ -52,7 +52,8 @@
       "run_mode_help": "Server run mode must be 'HYBRID', 'DISPATCH_ONLY', or 'GAME_ONLY'. Unable to start Grasscutter...",
       "create_resources": "Creating resources folder...",
       "resources_error": "Place a copy of 'BinOutput' and 'ExcelBinOutput' in the resources folder.",
-      "version": "Grasscutter version: %s-%s"
```

Comment:
Don't remove this

Reply:
What's wrong?

Reply:
Changing the substitution parameter count would normally be a problem but I'm not getting any hits for existing usage of this so it might be fine.

Reply:
I've traced it back to https://github.com/Grasscutters/Grasscutter/commit/edc75b2632318c99671cb089d6ed694750864c0c but it looks like the code that was referencing this translation string has since been replaced with https://github.com/mingjun97/Grasscutter/blob/2dc0120c81ff0c47af3f1e30069027d0622ea5c5/src/main/java/emu/grasscutter/Grasscutter.java#L84 so it's probably fine to repurpose this so long as the new version string is appropriate (I haven't checked that)

Reply:
The new version string is `1.1.2-dev` so it wouldn't even be correct to use %s-%s.

Reply:
> The new version string is `1.1.2-dev` so it wouldn't even be correct to use %s-%s.

The second %s is for the git hash so we can easily identify someone’s build (primarily for development but also useful in stable)

Reply:
> > The new version string is `1.1.2-dev` so it wouldn't even be correct to use %s-%s.
> 
> The second %s is for the git hash so we can easily identify someone’s build (primarily for development but also useful in stable)

Ohhhh sorry about that, I'll make sure to re-implement that as soon as I can.

Reply:
Yup, it works for me now.

Reply:
> > The new version string is `1.1.2-dev` so it wouldn't even be correct to use %s-%s.
> 
> The second %s is for the git hash so we can easily identify someone’s build (primarily for development but also useful in stable)

AFAICT it's only used to construct the version string, we may as well just have a single string for version and change the gradle script from `git rev-parse --verify --short HEAD` to `git describe --tags --dirty`